include/smack/Prelude.h

```diff
@@ -18,6 +18,24 @@ class Prelude {
   Prelude(SmackRep &rep) : rep(rep) {}
 
   std::string getPrelude();
+private:
+  void generateBinaryOps(std::stringstream& s, const unsigned size);
```

Comment:
Why did you turn all of these into member functions now?

Reply:
Some of these functions will use the `SmackRep` class which is only accessible being a member function.